src/client/Microsoft.Identity.Client/Internal/Broker/BrokerResponseConst.cs

```diff
@@ -44,5 +44,7 @@ internal static class BrokerResponseConst
 
         //Error codes returned from broker
         public const string NoTokenFound = "no_tokens_found";
+        public const string NoAccountFound = "no_account_found";
+        public const string InvalidRefreshToken = "Broker refresh token is invalid";
```

Comment:
not sure if this is legit, i used the error codes from the customer's screen shot.

Reply:
It's legit :( Both Android broker and B2C miss the point of "error code" and send full blown messages :(